target/linux/ramips/image/mt7621.mk

```diff
@@ -110,6 +110,11 @@ define Build/ubnt-erx-factory-image
 	fi
 endef
 
+define Build/ubootpad96
```

Comment:
`dlink-uimage` or such

Reply:
Please also take care of proper sorting when you update.

Reply:
Updated